jenkins-slaves/jenkins-slave-golang/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openshift/jenkins-slave-base-centos7:latest
+# FROM openshift/jenkins-slave-base-rhel7:latest
```

Comment:
https://github.com/redhat-cop/containers-quickstarts/pull/80/files#diff-b06b46547848dce57f3d821c78624575R21 will run this build with a RHEL base in OCP. There can be times where there is a material difference between centos and rhel, so the Dockerfile will work in one and not the other. I don't see anything here like that (e.g. EPEL is the biggie), but its good to know

Reply:
@sherl0cks  - yeah agreed the `FROM` is overwritten in OCP land when running a build and Im happy for that, aware of EPEL and such. I do find it useful to be able to build locally in Centos all the same as I don't have to mess around with access to rhel repos.

Reply:
@pabrahamsson  -  I could change it to to `FROM registry.access.redhat.com/<distro>`; that's no problem. This `FROM` block is only used for local building / testing, so wanted to keep it `centos` based. When you run the template in OCP land it gets overwritten by the RHEL Image as per @sherl0cks  comment

Reply:
I think that the proper pattern for `FROM` lines should be NOT to specifiy the registry. Allow the build environment to include approved registries in search paths. This allows for enterprise registries to be used as mirrors without code changes.

Reply:
Cool - so leave it as it is then @etsauer ? Can we get this puppy in so I can bring across the rest of the slaves I have outstanding on the `labs-ci-cd` repo then?

Reply:
I'm cool to leave it as is based on the comment from @etsauer 
For the sake of brevity, what controls the image for the slave build is https://github.com/redhat-cop/containers-quickstarts/pull/80/files#diff-b06b46547848dce57f3d821c78624575R60 rather than what @sherl0cks was referencing.
Unless I've gotten it all backwards we can remove the `jenkins-slave-base-rhel7` ImageStream (which is what was originally referenced).